v1/nix/modules/drvs/pillow/default.nix

```diff
@@ -47,12 +46,9 @@ in {
     ];
   };
 
-  mach-nix.pythonSources = config.deps.fetchPip {
-    inherit python;
-    name = config.name;
-    requirementsList = ["${config.name}==${config.version}"];
-    hash = "sha256-eS81pqSjU6mgBL6tXadSxkGdafsVFThByOQcOf8FkF0=";
+  mach-nix.pythonSources.fetch-pip = {
     maxDate = "2023-01-01";
+    requirementsList = ["${config.name}==${config.version}"];
     pipFlags = [
       "--no-binary"
```

Comment:
We now have an option for --no-binary

Reply:
Hm, I think we only have one for `onlyBinary`. We could add one for `onlySource` though.